src/app/components/MarkdownCards.tsx

```
@@ -0,0 +1,44 @@
+        <>
+            <aside className="warning">
+                <span>Warning:</span>
+                <div className="text_read">
+                    {children}
+                </div>
+            </aside>
+        </>
+    )
+}
+
+export const Error=({children}:Props)=>{
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)